pockemun/src/views/HomeView.vue

```diff
@@ -1,7 +1,91 @@
-<script setup lang="ts"></script>
-
 <template>
-  <main>
-    <div />
-  </main>
+  <div
```

Comment:
Maybe move this div to separate component (ex. PokemonBanner) to avoid having all this long ass link in the template? WDYT?